content/en/docs/reference/kubectl/generated/kubectl_taint/_index.md

```diff
@@ -30,6 +30,7 @@ Update the taints on one or more nodes.
   *  The value is optional. If given, it must begin with a letter or number, and may contain letters, numbers, hyphens, dots, and underscores, up to 63 characters.
   *  The effect must be NoSchedule, PreferNoSchedule or NoExecute.
   *  Currently taint can only apply to node.
+  *  Note that if you manually schedule a pod by defining a node name in your workload yaml spec file like a pod, deployments, etc. it will override the usual scheduling mechanism, hence, bypassing the taint.
```

Comment:
This doesn't belong here I'm afraid.

- the file is [autogenerated](https://kubernetes.io/docs/reference/kubectl/generated/kubectl_taint/#auto-generated-edit-notice)


instead, we can add a way to inject link to concept pages such as https://kubernetes.io/docs/concepts/scheduling-eviction/taint-and-toleration/
but, somebody needs to make that mechanism before we start to use it.

We should not make this change, as the autogeneration process will overwrite it.

Reply:
Unfortunately, the kind of fix I have in mind involves some web development work. Once that has happened (and _only_ when), then we could make the change on that concept page.

I think we should focus on other improvements that have a higher priority.

Reply:
Okay, what I had planned to do afterwards was specify it as a note on the concept page, at the end of the [Taint Nodes by Condition](https://kubernetes.io/docs/concepts/scheduling-eviction/taint-and-toleration/#taint-nodes-by-condition) sub-heading.

Example:
> **Note:** If you manually schedule a pod by defining a node name in your workload yaml spec file like a pod, deployments, etc. it will override the usual scheduling mechanism, hence, bypassing the taint.

Reply:
OK, that'd work.

Reply:
Awesome! [Done](https://github.com/kubernetes/website/pull/45384)